core/src/main/java/io/lionweb/lioncore/java/serialization/DeserializationStatus.java

```diff
@@ -22,13 +25,17 @@ class DeserializationStatus {
   final List<SerializedClassifierInstance> sortedList;
   final List<SerializedClassifierInstance> nodesToSort;
   final List<ProxyNode> proxies = new ArrayList<>();
-  private final AbstractSerialization serialization;
+  private LocalClassifierInstanceResolver proxiesInstanceResolver;
+  private ClassifierInstanceResolver complexiveInstanceResolver;
```

Comment:
what is "complexive"?

Reply:
Or do you (@ftomassetti) mean an English variant of “complessivo”?

Reply:
Exactly, you know Italian better than I know English :) I will look up the proper word. Maybe Overall?

Reply:
Ha, I wouldn't dare to claim that, but thanks for the compliment! :)

I was thinking "global".

Reply:
Renaming it as globalClassifierResolver and adding a comment